src/timeMachine/selectors/index.ts

```diff
@@ -65,6 +68,14 @@ export const getActiveQuery = (state: AppState): DashboardDraftQuery => {
   return draftQueries[activeQueryIndex]
 }
 
+export const getActiveQueryIndex = (state: AppState): number => {
+  const activeTimeMachine = getActiveTimeMachine(state)
+  if (activeTimeMachine) {
+    return activeTimeMachine.activeQueryIndex
+  }
+  return 0
```

Comment:
is the active query index always going to default to 0? are the instances when it might be `undefined` or some other value?

Reply:
theoretically yes, practically no. As in there's no guards against it in the system, but timemachine always starts up with one in the default state and doesn't allow you to delete all of the queries. seems flimsy, but it an ism that holds up a bunch of other logic as well